src/main/resources/com/google/api/codegen/ruby/test.snip

```diff
@@ -0,0 +1,166 @@
+  # How should we expose it?
+  require "google/cloud/language/v1beta1/language_service_api"
+  # include Google::Cloud::Language::V1beta1
+  include {@util.getStubs(testClass.service).get(0).namespace}
+
+  # describe LanguageServiceApi do
+  describe {@testClass.apiClassName} do
+
+    @let gRPCClass = util.getStubs(testClass.service).get(0).grpcClientTypeName
+      def stub_grpc_client
+        mock_client = Minitest::Mock.new
+
+        @# Implement "method" method for the mock.
+        def mock_client.method(name_symbol)
+          @# We return a lambda (an instance of Proc) instead of a Method, because
+          @# the latter can't be easily constructed on the fly. In practice, most
+          @# operations performed on Methods work on Procs. The differences are:
+          @# Proc implements: :yield, :lambda?, :binding
+          @# Method implements: :receiver, :name, :original_name, :owner, :unbind,
+          @#                    and :super_method
+          lambda { |*args| public_send(name_symbol, *args) }
+        end
+
+        # TODO(jcanizales): Instead of mock_client, pass a block that verifies
+        # the host string and that the 2nd argument isn't
+        # :this_channel_is_insecure.
+        # LanguageService::Stub.stub(:new, mock_client) { yield(mock_client) }
+        {@gRPCClass}::Stub.stub(:new, mock_client) { yield(mock_client) }
+      end
+
+      @join testCase : testClass.testCases
+        @let methodName = testCase.surfaceMethodName
+          # TODO(jcanizales): Undo this join-if hack.
+          @join method : util.getSupportedMethodsV2(testClass.service)
+            @if util.upperCamelToLowerUnderscore(method.getSimpleName()).equals(methodName)
+              @let methodView = util.getMethodView(testClass.service, method)
+                # describe "#annotate_text" do
+                describe "#{@methodName}" do
+                  it "forwards calls to {@gRPCClass}::Stub" do
+                    stub_grpc_client do |mock_client|
+                      # TODO(jcanizales): Initialize enums with the second value, strings
+                      # with "sample", arrays with 1 element, etc. Do the same with the
+                      # mocked response.
+                      # TODO(jcanizales): Instead of aliasing all used types,
+                      # include Google::Cloud::Language::V1beta1 in the sample.
+                      {@sampleCodeAliases(methodView)}
+
+                      # expected_request = AnnotateTextRequest.new(            
+                      expected_request = {@testCase.requestTypeName}.new(
+                        {@initRequest(methodView.initCode)}
+                      )
+
+                      # mocked_response = AnnotateTextResponse.new
+                      mocked_response = {@testCase.responseTypeName}.new
+                      # TODO(jcanizales): testCase.mockResponse.initCode
+
+                      mock_client.expect(
```

Comment:
This only tests for the happy cases. Probably add a TODO to add test for exceptions.